src/earn/EarnEnterAmount.test.tsx

```diff
@@ -234,39 +208,24 @@ describe('EarnEnterAmount', () => {
     })
     await waitFor(() => expect(getByText('earnFlow.depositBottomSheet.title')).toBeVisible())
   })
-  it('should handle navigating to the add crypto bottom sheet', async () => {
+  it('should show a warning and not allow the user to continue if they input too high of a number', async () => {
```

Comment:
```suggestion
  it('should show a warning and not allow the user to continue if they input amount greater than balance', async () => {
```
to remove any ambiguity